bazel/scripts/run_integ_tests.sh

```diff
@@ -348,6 +353,7 @@ FAILED_LIST=()
 FAILED_LIST_FILE="/tmp/last_failed_integration_tests.txt"
 MERGED_REPORT_FOLDER="/var/tmp/test_results"
 INTEGTEST_REPORT_FOLDER="${MERGED_REPORT_FOLDER}/integtest_reports"
+PROFILE="--profile=bazel_profile_lte_integ_tests"
```

Comment:
When running inside the repo this creates the profile inside the repo, which is not nice. The bazel default would usually be `/var/tmp/bazel/command.profile.gz`, but this depends on the output base. But we could at least to into `/var/tmp` and maybe print the profile location at the end of the script with the summary?